src/main/kotlin/ro/hibyte/betting/controller/EventTemplateController.kt

```diff
@@ -29,8 +30,12 @@ class EventTemplateController(private val eventTemplateService: EventTemplateSer
         eventTemplateService.getById(id)
 
     @GetMapping
-    fun getAll(): List<EventTemplate> =
-        eventTemplateService.getAll()
+    fun getAll(@RequestParam name: String?): List<EventTemplate> =
```

Comment:
You can do this instead of using an `if`

```kotlin
    @GetMapping
    fun getAll(@RequestParam name: String?): List<EventTemplate> =
        name?.let { eventTemplateService.searchByName(it) } ?: eventTemplateService.getAll()
```